pkg/gather/clusterconfig/0_gatherer.go

```diff
@@ -2,290 +2,92 @@ package clusterconfig
 
```

Comment:
Let's rename this file please. `0_gatherer` is a bit meaningless IMHO :). We can go with `clusterconfig.go` (or `clusterconfig_gatherer.go`) or any better suggestions?